dbt_project.yml

```diff
@@ -24,13 +24,19 @@ clean-targets:         # directories to be removed by `dbt clean`
     - "target"
     - "dbt_modules"
 
+vars:
+  customer: customer_a
+  target_snapshot_schema: "{{ target.schema if target.name == 'dev' else 'prod_snapshot' }}"
```

Comment:
@patkearns10 is this the recommended way to handle schemas for snapshots in dev? I recently came across an old post about it but I didn't know what we usually advise

Reply:
Yeah, probably not! 

The docs state that you should treat snapshots like source data, which would make this totally unnecessary, because the raw data in dev should be the same as raw data in prod.

I wrote this in response to someone asking about using a snapshot further to the right in their dag, and not wanting to mess it up if they only run part of the data in development. I think I also used a variable rather than the preferred method below, so that it could be iterable in a job command. Anyways, I'll remove that! 

https://docs.getdbt.com/reference/snapshot-configs#apply-the-target_schema-configuration-to-all-snapshots-in-your-project
(It would probably be better to add a `snapshot` block to the `dbt_project.yml` with that overwrite.)